pkgs/web_socket/pubspec.yaml

```diff
@@ -0,0 +1,11 @@
+name: web_socket
+description: "TODO: enter a descirption here"
+version: 1.0.0
```

Comment:
I would start without a version, and make it explicit when it gets into a publishable state.

```suggestion
publish_to: none
```